### ISSUE TITLE
Error in output image

### ISSUE THREAD

**Issue by Arun-1997** [matterport/Mask_RCNN#1164]
I trained a custom dataset with 39 images for train and 5 images for val. The system I use is a windows 10 OS with 16 GB Memory with Nvidia Geforce 1GB GPU. When I run my detection file, the output is displayed thus: 
![out1](https://user-images.githubusercontent.com/22298702/49579827-59538080-f973-11e8-83b4-7bbcb6b33e82.png)

but the required mask needs to be thus: 
![inkedtest11_li](https://user-images.githubusercontent.com/22298702/49580010-ded73080-f973-11e8-8c39-77c631c40fab.jpg)

Can anyone specify as to where the error or issue occurs? The loss value is well reduced and has a value of 0.035.

**Comment by stygian2a** [comment]
Can you show us one of you  mask label please ?

**Comment by Arun-1997** [comment]
hi stygian2a, what is a mask label?  I am new to tensorflow. so sorry if the question is amateur
